client/integration-tests/pom.xml

```
@@ -44,6 +44,7 @@
     <module>config-key</module>
     <module>github</module>
     <module>without-oidc</module>
+      <module>serializable-model</module>
```

Please fix the indentation.

```suggestion
    <module>serializable-model</module>
```